### DISCUSSION TITLE
How to use it for unpair data？

### DISCUSSION THREAD

**Discussion by wangnanv5** [TorchIO-project/torchio#1226] [Q&A]
hello, torchio in my segmentation assignment acted pretty well,but i want know how to use it in unpair task,like cycyleGan task, i want  to load different image pairs each time,rather than having fixed image-label pairs like  segmentation assignment.
Looking forward to your reply!

**Comment by romainVala** [comment]
Hi
not sure to understand what you are looking for.
The whole point of torchio is to identically transform all images assign to a subject

but you can easily achieve all you want with the `include`/`exclude` argument of the transform

imagine you construct a subject with the same image in two subject's keys
```
i = torchio.ScalarImage('some_path')
suj = torchio.Subject({'image1':i, 'image2':i})
```

then you can add any transform on image2 only
```
my_transform = torchio.RandomAffine(include='image2')
sujt = my_transform(suj)

```
then only image2 has been transformed
